### ISSUE TITLE
Incorrect upload link on the main page

### ISSUE THREAD

**Issue by piotr-jagiello** [jhaals/yopass#1127]
Hi,
in contrast to the other issue I submitted, this one refers to the build tagged with the latest release (11.2.0), since due to that other bug I can't deploy the master branch.

Deployment method is AWS Lambda, but the issue seems to be with the frontend. The setup I have hosts Yopass frontend under something like "ots.example.com", and the frontend is built like so, from the website directory:
```
yarn install
PUBLIC_URL="https://ots.example.com" REACT_APP_BACKEND_URL="https://otsapi.example.com" yarn build
```
(this is used in a CI, with the `node:lts-bullseye` Docker container).

This seems to result in the upload button that links to this address:
`https://ots.example.com/#/https://ots.example.com/upload`

which just doesn't work, and actually breaks the app (the portion of the site for submitting secrets isn't shown anymore).
Clicking at the button again continues the weirdness:
`https://ots.example.com/#/https://ots.example.com/https://ots.example.com/`

Clicking the button more just appends more absolute urls at the end.
Visiting `https://ots.example.com/#/upload` directly seems to work fine.

**Comment by jhaals** [comment]
Thanks for reporting! Should be fixed in release 11.3.0. Please reopen this issue if the problem remains.

**Comment by piotr-jagiello** [comment]
Thanks for fixing this! I can confirm that frontend 11.3.0 works fine now.
